google-cloud-firestore/clirr-ignored-differences.xml

```diff
@@ -314,4 +314,19 @@
     <method>com.google.cloud.firestore.Query getQuery()</method>
   </difference>
 
+    <!-- Relocate constants in TraceUtil -->
+  <difference>
+    <differenceType>6011</differenceType>
+    <className>com/google/cloud/firestore/telemetry/TraceUtil</className>
+    <field>SPAN_NAME_*</field>
+  </difference>
+
+  <!-- Change the parameter type of internalStream function -->
+  <difference>
```

Comment:
waiting for https://github.com/googleapis/java-firestore/pull/1896 to lift the clirr alert

Reply:
I believe the decision for #1896 was to use `internalApi` rather than making the class package-private. So we'll need to keep this clirr change as well.

Reply:
they are changing the `internalStream` to default from `protected`. So it would affect the public access of that method, which could remove the need to add clirr for that functions parameter change.